packages/babel/src/transformation/file/options/config.json

```diff
@@ -240,6 +236,12 @@
     "description": "set `sources[0]` on returned source map"
   },
 
+  "inputSourceMap": {
+    "type": "boolean",
+    "description": "use existing source map for source map generation - only works with inline source maps",
```

Comment:
Thank you for the explanation; much appreciated.  I saw the population from the comment and made the wrong assumption that this should be a boolean flag to enable that to happen.  I overlooked that this option expects an object.  I'll use the `babel` API directly to achieve what I need.  Thanks!